.github/workflows/versioned-docs.yml

```
@@ -0,0 +1,133 @@
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
+        
+  notify-discord:
+    needs: [build-versioned-docs]
+    name: Notify Discord
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)